meson_options.txt

```diff
@@ -27,6 +27,7 @@ option('plugin_modem_manager', type : 'boolean', value : false, description : 'e
 option('plugin_msr', type : 'boolean', value : true, description : 'enable MSR support')
 option('plugin_flashrom', type : 'boolean', value : false, description : 'enable libflashrom support')
 option('plugin_platform_integrity', type : 'boolean', value : false, description : 'enable platform integrity support')
+option('qubes_enable', type : 'boolean', value : false, description : 'build packages for Qubes OS')
```

Comment:
I think `qubes` is fine; the _enable_ is implied.

Reply:
Changed.